src/daemon/common.h

```diff
@@ -384,4 +384,8 @@ void strarray_free(char **array, size_t array_len);
 int check_capability(int arg);
 #endif /* HAVE_SYS_CAPABILITY_H */
 
+#ifndef PRIsz
+#define PRIsz PRIu64
```

Comment:
Is there a reason you didn't use the [latest version](https://github.com/Stackdriver/collectd/blob/windows-fixes/src/daemon/common.h#L390) of those changes?

Reply:
I left out the Windows bits, since it doesn't currently run on Windows, so I couldn't make sure it all worked.

If you want, I can add them in, since it is just copying and pasting over.

Reply:
Nah, I was proposing to define `PRIsz` in terms of what works on Linux for now, so you could add additional cases later.